Formula/lib/libgda.rb

```diff
@@ -29,7 +29,7 @@ class Libgda < Formula
   depends_on "glib"
   depends_on "iso-codes"
```

Comment:
No linkage for `iso-codes`; wonder if it's needed.

Reply:
It provides an XML file that is read at runtime
https://gitlab.gnome.org/GNOME/libgda/-/blob/LIBGDA_6_0_0/libgda-ui/gdaui-init.c?ref_type=tags#L695